README.md

```diff
@@ -22,10 +22,17 @@ A data-driven `UICollectionView` framework for building fast and flexible lists.
 
 `IGListKit` is built and maintained by [Instagram engineering](https://engineering.instagram.com/), using the open source version for the Instagram app.
 
+## Requirements
+
+- Xcode 8.0+
+- iOS 8.0+
+- Interoperability with Swift 3.0+
```

Comment:
Yeah -- everything should actually work here 😄 

However, all of our examples are Swift 3 and we're only testing on Swift 3. So I think we should only claim support for Swift 3+

cc @rnystrom - thoughts?